src/main/java/org/opensearch/flowframework/model/WorkflowNode.java

```diff
@@ -50,7 +50,7 @@ public class WorkflowNode implements ToXContentObject {
     /** The field defining the timeout value for this node */
     public static final String NODE_TIMEOUT_FIELD = "node_timeout";
     /** The default timeout value if the template doesn't override it */
-    public static final String NODE_TIMEOUT_DEFAULT_VALUE = "15s";
+    public static final String NODE_TIMEOUT_DEFAULT_VALUE = "10s";
```

Comment:
Mentioned this as an aside [here](https://github.com/opensearch-project/flow-framework/pull/317/files#r1436647786), this is OK for this PR but we might want to consider using one of the settings for this instead of hardcoding it.